### PR TITLE
[win32] fix mirror location after 383c3dd71d0c71555b27a3e70724d8efbe9…

### PR DESCRIPTION
**Pull request by MartijnKaijser**
…14f6f

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@ace20022 @afedchin

File: project/BuildDependencies/DownloadMingwBuildEnv.bat [hunk 1]
```diff
@@ -518,7 +518,7 @@ if %build32%==no GOTO loginProfile64
     )
 
 :loadlocals32
-set pkgbaseurl=%KODI_MIRROR%/locals
+set pkgbaseurl=%KODI_MIRROR_MSYS2%/locals
```
Comment (afedchin) [inline project/BuildDependencies/DownloadMingwBuildEnv.bat]: please add the following line before L521
```
if "%KODI_MIRROR_MSYS2%" == "" goto end
```

**Comment by afedchin** [comment]
looks good except minor comment. thank you!

**Comment by MartijnKaijser** [comment]
saw you just pushing the same to your repo :)

**Comment by afedchin** [comment]
You was faster :)

**Pull request merged**